packages/spacecat-shared-data-access/test/unit/service/import-job/index.test.js

```diff
@@ -37,14 +38,18 @@ describe('Import Job Tests', () => {
 
     beforeEach(() => {
       mockDynamoClient = {
-        getItem: sinon.stub().resolves([]),
+        getItem: sinon.stub().resolves(),
         query: sinon.stub().resolves(null),
         putItem: sinon.stub().resolves(),
       };
       mockLog = {
         log: sinon.stub(),
```

Comment:
You can also consider using `console` here instead of `sinon.stub()`, which can help when writing & debugging tests.

Reply:
Done